Manito/Manito/Screens/Login/LoginViewController.swift

```diff
@@ -6,157 +6,106 @@
 //
 
 import AuthenticationServices
+import Combine
 import UIKit
 
 import SnapKit
 
-final class LoginViewController: UIViewController, BaseViewControllerType, Navigationable {
+final class LoginViewController: UIViewController, Navigationable {
 
     // MARK: - ui component
 
-    private let logoImageView: UIImageView = UIImageView(image: UIImage.Image.appIcon)
-    private let logoTextImageView: UIImageView = UIImageView(image: UIImage.Image.textLogo)
-    private let appleLoginButton: ASAuthorizationAppleIDButton = {
-        let button = ASAuthorizationAppleIDButton(type: .signIn, style: .white)
-        button.cornerRadius = 25
-        return button
-    }()
+    private let loginView: LoginView = LoginView()
     
     // MARK: - property
     
-    private let loginRepository: LoginRepository = LoginRepositoryImpl()
+    private var cancellable = Set<AnyCancellable>()
+    private let loginViewModel: LoginViewModel
     
     // MARK: - init
     
+    init(viewModel: LoginViewModel) {
+        self.loginViewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
```

Comment:
p1;

viewModel 도 추상화 부탁드릴게요!
```suggestion
    private let loginViewModel: any BaseViewModelTpye
       
    // MARK: - init
    
    init(viewModel: any BaseViewModelType) {
        self.loginViewModel = viewModel
        super.init(nibName: nil, bundle: nil)
    }
```

Reply:
[20ae10e](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/20ae10e547832dc30a44171c6796afb132fa2a7d)
수정하였습니다.